### PR TITLE
#93 - refactor

### PR DESCRIPTION
**Pull request by boltonlin**
[Ticket](https://trello.com/c/e3JzO82Y/93-refactor)
- moved shared components from `helpers/` to `components/shared`
- renamed widget directories to PascalCase
- minor config changes to facilitate testing
- moved tests out of `src/` into `/__tests__/`

### CODE REVIEW

**Comment by boltonlin** [comment]
closing this, sorry. minor bug, investigating
image

**Comment by boltonlin** [comment]
closing this, sorry. minor bug, investigating
image

File: client/src/components/Questions/styles/questions.css [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-  margin-bottom: 5em;
-  max-width: 1000px;
-}
+/* GENERAL TAGS */
 
-.qa.question {
-  display: flex;
-  align-items: baseline;
-  justify-content: space-between;
+div#questions-widget {
```
Comment (jerrodvarney) [inline client/src/components/Questions/styles/questions.css]: I don't recommend using the div tag here. The id tag is as specific as it gets and wont overwrite anything else.

**Review by jerrodvarney — CHANGES_REQUESTED** [review]
Just the one css comment.

**Review by jerrodvarney — APPROVED** [review]
Looks good!

**Pull request merged**